.github/workflows/build.yml

```diff
@@ -57,9 +50,16 @@ jobs:
               3.1.x
               5.0.x
               6.0.x
-            
-        - name: Run build script
-          run: ${{ matrix.build_script }}
+
+        - name: Build
+          run: dotnet build SentryNoSamples.slnf -c Release /p:CopyLocalLockFileAssemblies=true
+
+        - name: Test
+          run: dotnet test SentryNoSamples.slnf -c Release --no-build -l "GitHubActions;report-warnings=false"
+
+        - name: Pack
+          if: matrix.os == 'windows-2022'
+          run: dotnet pack SentryNoSamples.slnf -c Release --no-build /p:ContinuousIntegrationBuild=true
 
         - name: Publish coverage report
           uses: codecov/codecov-action@v1
```

Comment:
I thought we were no longer doing code coverage, so should we remove this step?

Reply:
https://github.com/getsentry/sentry-dotnet/pull/1588